lib/pbench/server/auth/auth.py

```diff
@@ -155,21 +114,21 @@ def verify_auth(auth_token: str) -> Optional[User]:
     return user
 
 
-class TokenState(enum.Enum):
-    """The state of a token once decoded."""
+def verify_auth_api_key(api_key: str) -> Optional[User]:
+    """Validates the api_key of the current request.
 
-    INVALID = enum.auto()
-    EXPIRED = enum.auto()
-    VERIFIED = enum.auto()
+    Tries to validate the api_key as if it was generated by the Pbench server .
```

Comment:
But _it is_ generated by the pbench server

Reply:
😆 

If you're actually objecting, @npalaska, you should suggest verbiage which would be acceptable.  😉